### PR TITLE
Add Japanese docs

### PR DESCRIPTION
**Pull request by eltociear**
### PR types

 Others
### PR changes

Docs 
### Description

Added more Japanese translated documents.

### CODE REVIEW

File: CODE_OF_CONDUCT_ja.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# 投稿者規約 行動規範
```
Comment (jzhang533) [inline CODE_OF_CONDUCT_ja.md]: For this code of conduct document, we are currently in the process of discussing an update to a more recent version. We have chosen to adopt version 1.4 of the [Contributor Covenant](https://www.contributor-covenant.org/). Due to historical reasons, the Simplified Chinese translation has not been satisfactory.

The most recent iteration is version 2.1, which includes an official translation in Simplified Chinese. However, it appears that version 2.1 has not yet been translated into Japanese, although they have established a mature [process for accepting translations](https://github.com/EthicalSource/contributor_covenant/tree/release#translating).

In the context of the PaddlePaddle community, I believe the optimal approach would involve adopting translations from the latest version of the [Contributor Covenant](https://www.contributor-covenant.org/). Nonetheless, it's essential for us to reach a consensus before proceeding.

File: SECURITY_ja.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# PaddlePaddle を安全に使用する
```
Comment (jzhang533) [inline SECURITY_ja.md:1]: Although I can understand your Japanese translation to the security documents, my Japanese is not good enough to give review comments. 

But I think we can accept your translation, this is a great addition to PaddlePaddle community.

I will merge your PR, after you remove the translation to Code of Conduct.

Comment (eltociear) [reply SECURITY_ja.md:1]: @jzhang533 
Thank you for your confirmation.
Code of Conduct file has been removed.

**Review by jzhang533 — APPROVED** [review]
LGTM

**Pull request merged**